_posts/2021-08-16-connector-table-sql-api-part1.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: post
+title:  "Implementing a custom source connector for Table API and SQL - Part One "
+date: 2021-08-18T00:00:00.000Z
+authors:
+- Ingo Buerk:
+  name: "Ingo Buerk"
+excerpt: 
+---
+
+{% toc %}
+
+# Introduction
```

Comment:
As a general remark, I know that in the documentation we use "you" rather than "we". This is of course a completely biased opinion, but in a blog post like this "we" reads much more inclusive to me compared to "You will be doing XYZ". I'm wondering how others feel about this?

Reply:
I based the decision on some previous experiences at other companies and also on https://www.gatsbyjs.com/contributing/gatsby-style-guide/#word-choice but I am open to a discussion about it!

Reply:
It does seem to be a common way of doing it, so I guess "it's just me". :-)